docs/src/pages/react-native.md

```diff
@@ -3,8 +3,78 @@ id: react-native
 title: React Native
 ---
 
-React Query is designed to work out of the box with React Native, with the exception of the devtools, which are only supported with React DOM at this time. 
+React Query is designed to work out of the box with React Native, with the exception of the devtools, which are only supported with React DOM at this time.
 
 There is a 3rd party [Flipper](https://fbflipper.com/docs/getting-started/react-native/) plugin which you can try: https://github.com/bgaleotti/react-query-native-devtools
 
 If you would like to help us make the built-in devtools platform agnostic, please let us know!
+
+## Online status management
+
+React Query already supports auto refetch on reconnect in web browser.
+To add this behavior in React Native you have to use React Query `onlineManager` as in the example below:
+
+```ts
+import React from 'react'
+import NetInfo from '@react-native-community/netinfo'
+import { onlineManager } from 'react-query'
+import { Platform } from 'react-native'
+
+export function useOnlineManager() {
+  React.useEffect(() => {
+    if (Platform.OS !== 'web') {
+      return NetInfo.addEventListener(state => {
+        onlineManager.setOnline(
+          state.isConnected != null &&
+            state.isConnected &&
+            Boolean(state.isInternetReachable)
+        )
+      })
+    }
+  }, [])
+}
+```
+
+## Refetch on App focus
+
+In React Native you have to use React Query `focusManager` to refetch when the App is focused.
+You can use 'react-native-appstate-hook' to be notified when the App state has changed.
+
+```ts
+import { focusManager } from 'react-query'
+import useAppState from 'react-native-appstate-hook'
+
+function onAppStateChange(status: AppStateStatus) {
+  if (Platform.OS !== 'web') {
+    focusManager.setFocused(status === 'active')
```

Comment:
same question, see [focusManager.setEventListener](https://react-query.tanstack.com/reference/focusManager#focusmanagerseteventlistener)

Reply:
The example I provide is a React Native based solution using `react-native-appstate-hook` compared to the existing `setEventListener` documentation using `window`.